### PR TITLE
[orchagent]: Export platform information before orchagent run.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
Export platform information in environment to be able to write vendor
specific code.

### CODE REVIEW

File: dockers/docker-orchagent/orchagent.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Export platform information. Required to be able to write
+# vendor specific code.
+export platform=`sonic-cfggen -m /etc/sonic/minigraph.xml -v onie_switch_asic`
```
Comment (lguohan) [inline dockers/docker-orchagent/orchagent.sh]: can you use the asic_type?

Comment (oleksandrivantsiv) [inline dockers/docker-orchagent/orchagent.sh]: It has different output:
```
root@arc-switch1026:/home/admin# sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type
mellanox
root@arc-switch1026:/home/admin# sonic-cfggen -m /etc/sonic/minigraph.xml -v onie_switch_asic
mlnx
```
Everywere in swss code **mlnx** is expected.

Comment (oleksandrivantsiv) [inline dockers/docker-orchagent/orchagent.sh]: We need this change ASAP to be able to run images on MLNX platforms. If it is critical please let me know and I'll change code here and in swss to use **asic_type**.

Comment (oleksandrivantsiv) [inline dockers/docker-orchagent/orchagent.sh]: Definition of platform:
https://github.com/Azure/sonic-swss/blob/27c49a0bf3a71556b553aa30fe2a7c1cdb996142/orchagent/copporch.cpp#L16
https://github.com/Azure/sonic-swss/blob/476772db125f3791a986d8c7d6f89b6d812e6234/orchagent/routeorch.cpp#L17
Usage:
https://github.com/Azure/sonic-swss/blob/27c49a0bf3a71556b553aa30fe2a7c1cdb996142/orchagent/copporch.cpp#L117
https://github.com/Azure/sonic-swss/blob/476772db125f3791a986d8c7d6f89b6d812e6234/orchagent/routeorch.cpp#L51

Comment (stcheng) [inline dockers/docker-orchagent/orchagent.sh]: Hi @oleksandrivantsiv , Could you create a pull request to switch the code in swss to use mellanox instead? I think that could be a cleaner solution. After that we could clean up code here.

**Pull request merged**